LICENSE.md

```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2019 Digital Bazaar, Inc.
```

Comment:
```suggestion
Copyright (c) 2011-2020 Digital Bazaar, Inc.
```

Reply:
Addressed (the diff was outdated) https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/ba56da9231789067b9e3b3c26ae560a452a64c85